src/context/NotificationContext.tsx

```diff
@@ -0,0 +1,45 @@
+// Copyright 2019-2020 @Premiurly/polkassembly authors & contributors
```

Comment:
This file is not required anymore, please use the queueNotification.tsx file directly